modules/localgov_workflows_notifications/localgov_workflows_notifications.module

```diff
@@ -88,11 +88,11 @@ function localgov_workflows_notifications_node_form_validate($form, &$form_state
   $values = [];
   foreach ($localgov_service_contacts as $value) {
     if (is_array($value) && !empty($value['target_id'])) {
-      if (in_array($value['target_id'], $values)) {
+      if (in_array((string) $value['target_id'], $values, TRUE)) {
```

Comment:
Sometimes `target_id` is a string and then sometimes it's an integer this makes sure the `target_id` is always a string when checking if it is in the array.

When building the `values` array we also make sure we are always dealing with strings.